src/opts/container.rs

```diff
@@ -331,6 +331,40 @@ impl ToString for PublishPort {
     }
 }
 
+pub enum IpcMode {
+    None,
+    Private,
+    Shareable,
+    Container(String),
+    Host,
+}
```

Comment:
Might be helpful to have some comments on here. Taken from API reference https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerCreate

For the enum could be: `IPC sharing mode for the container`

And for variants:

    "none": own private IPC namespace, with /dev/shm not mounted
    "private": own private IPC namespace
    "shareable": own private IPC namespace, with a possibility to share it with other containers
    "container:<name|id>": join another (shareable) container's IPC namespace
    "host": use the host system's IPC namespace

Reply:
Okok, will do.

I should really get more used to documenting stuff. It always snowballs down the road for me.